Description: Adjust to Debian installation
 This patch adjusts several files to the Debian specific installation.
Author: Roland Stigge <stigge@antcom.de>

--- latex2html-2008-debian1.orig/config/install.pl
+++ latex2html-2008-debian1/config/install.pl
@@ -235,7 +235,7 @@ my %Install_items = (
   'styles'            => 'lib,recurse',
   # MRO: Tests are for pre-installation testing
   #'tests'             => 'lib,recurse',
-  'texinputs'         => 'lib,recurse',
+  #'texinputs'         => 'lib,recurse',
   #'texlive.pm'        => 'lib',
   'versions'          => 'lib,recurse',
   #'wrapper'           => 'lib,recurse',
@@ -372,7 +372,7 @@ foreach $item (sort keys %Install_items)
 
 if($cfg{TEXPATH}) {
   print "\nNote: trying to install LaTeX2HTML style files in TeX directory tree\n     ($cfg{TEXPATH})\n";
-  unless(mkpath($cfg{TEXPATH})) {
+  unless($ENV{'INSTALLING_DEBIAN'} || mkpath($cfg{TEXPATH})) {
   #my $testpath = $cfg{TEXPATH}; # to strip (latex2)html
   #$testpath =~ s/[$dd$dd][^$dd$dd]*$//;
   #if((-d $cfg{TEXPATH} && !-w _) || (-d $testpath && !-w _)) {
@@ -394,7 +394,7 @@ if($cfg{TEXPATH}) {
         }
       }
       closedir(DIR);
-      if($ok && $cfg{MKTEXLSR}) {
+      if($ok && $cfg{MKTEXLSR} && ! $ENV{'INSTALLING_DEBIAN'}) {
         print "Info: Running $cfg{MKTEXLSR} to rebuild ls-R database...\n";
         system($cfg{MKTEXLSR});
       }

--- latex2html-2008-debian1.orig/makeseg/makeseg
+++ latex2html-2008-debian1/makeseg/makeseg
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl  
+#!/usr/bin/perl  
 
 # Check options and load configuration files
 &init;
@@ -17,23 +17,23 @@
 # config file settings therefore override anything set here
 
 # Location to look for L2H-supported LaTeX styles
-$LATEX2HTMLSTYLES="/usr/local/lib/texmf/latex2html/styles" 
+$LATEX2HTMLSTYLES="/usr/share/texmf/latex2html/styles" 
     unless $LATEX2HTMLSTYLES;
 
 # Location for TeX style files etc.
 # $TEXINPUTS is read from the environment if not explicitly supplied
 $TEXINPUTS=$ENV{'TEXINPUTS'} unless $TEXINPUTS;
-$TEXINPUTS=".:/usr/local/lib/texmf/tex//" unless $TEXINPUTS;
+$TEXINPUTS=".:/usr/share/texmf/tex//" unless $TEXINPUTS;
 
 # Location for BibTeX Bibliography databases
 # $BIBINPUTS is read from the environment if not explicitly supplied
 $BIBINPUTS=$ENV{'BIBINPUTS'} unless $BIBINPUTS;
-$BIBINPUTS=".:/usr/local/lib/texmf/bibtex/bib/" unless $BIBINPUTS;
+$BIBINPUTS=".:/usr/share/texmf/bibtex/bib/" unless $BIBINPUTS;
 
 # Location for BibTeX Bibliography styles (in addition to $TEXINPUTS)
 # $BSTINPUTS is read from the environment if not explicitly supplied
 $BSTINPUTS=$ENV{'BSTINPUTS'} unless $BSTINPUTS;
-$BSTINPUTS="/usr/local/lib/texmf/bibtex/bst/" unless $BSTINPUTS;
+$BSTINPUTS="/usr/share/texmf/bibtex/bst/" unless $BSTINPUTS;
 
 # The lowest sectioning command at which to create segments
 # corresponds to -l option
